docs/_source/components-overview/pop_syn/population_params.rst

```diff
@@ -323,23 +562,37 @@ It's possible to add custom hooks to the simulation steps.
 A few example hooks are provides: ``TimingHooks`` and ``StepNamesHooks`` (See :ref:`Custom Hooks <custom-hooks>` for more details)
 
 .. list-table::
-   :widths: 50 50
-   :header-rows: 1
+  :widths: 10 80 10
+  :class: population-params-table
+  :header-rows: 1
 
-   * - Parameter
-     - Description/Value
-   * - `import`
-     - ['posydon.binary_evol.simulationproperties', 'TimingHooks']
-   * - `absolute_import_1`
-     - None
-   * - `kwargs_1`
-     - {}
-   * - `import`
-     - ['posydon.binary_evol.simulationproperties', 'StepNamesHooks']
-   * - `absolute_import_2`
-     - None
-   * - `kwargs_2`
-     - {}
+  * - Parameter
+    - Description
+    - Default Value
+
+  * - ``import``
```

Comment:
The imports for the hooks have numbers: import -> import_1 (later import -> import_2)

I'm not sure whether we have a fixed number of hooks or whether this is flexible by having the combination of `import_*`, `absolute_import_*`, and `kwargs_*` as often as we want. In the case it is flexible, I'd suggest to update the docs to represent that.